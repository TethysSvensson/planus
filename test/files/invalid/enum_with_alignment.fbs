enum A: uint8 (force_align: 1) { X, Y }
