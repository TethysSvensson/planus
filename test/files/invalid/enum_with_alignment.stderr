error: Metadata attribute does not make sense on enums
  ┌─ test/files/invalid/enum_with_alignment.fbs:1:16
  │
1 │ enum A: uint8 (force_align: 1) { X, Y }
  │                ^^^^^^^^^^^^^^

